go.sum

```diff
@@ -0,0 +1,116 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I believe this file is autogenerated when dependencies are installed using "go mod tidy" so you probably shouldn't push it to GitHub.

Reply:
Really good point, sorry I missed that. This is fine as it is then!